yesod-persistent/Yesod/Persist/Core.hs

```diff
@@ -163,7 +165,30 @@ getBy404 key = do
         Nothing -> notFound'
         Just res -> return res
 
+-- | Create a new record in the database, returning an automatically
+-- created key, or raise a 400 bad request if a uniqueness constraint
+-- is violated.
```

Comment:
Can you do a minor version bump, update the changelog, and add `@since` comments to these haddocks?